src/components/Header/HeaderNav/HeaderNav.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import './HeaderNav.css';
```

Comment:
What do you think about removing these empty files? In general, for these components, we will try to avoid local styles and _we will use _Doppler UI System_ ones, also, for report specific components we decided using a different approach: _styled-components_ (if I am not wrong)

Reply:
I agree with this, if we are not using it right now, remove it please.

Reply:
> So what you say: Use the library and call components with customized style?

Perdón, mi inglés es bastante malo, así que mejor switcheo.

Lo que propongo es que para componentes de este tipo (header, footer, botones standard, fuentes, etc), usemos _la librería_ (_Doppler UI System_) que va a estar compartida con la aplicación _legacy_ de Doppler. Entonces, por ejemplo, _HeaderNav.css_ no sería necesario.

Por otro lado, para los componentes de React específicos de esta aplicación web [habíamos convenido utilizar _styled-components_](https://github.com/FromDoppler/doppler-webapp/issues/6#issuecomment-447009854), serán estilos más relacionados con comportamiento que, por ejemplo, con imágen de marca.

Reply:
@GusBaamonde respondí a tu pregunta?

Reply:
Los borro y en el caso de necesitar se agregan. Gracias!